mocks/github.com/docker/machine/libmachine_mocks/API.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2018 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by mockery v1.0.0. DO NOT EDIT.
+package libmachine_mocks
```

Comment:
nit: underscores are frowned upon in Go package names

Reply:
alright, I'll change that, thank you